connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -43,7 +43,9 @@
 
     public static final String CONNECTOR_CLASS_CONFIG = "connector.class";
```

Comment:
I know there's a backwards compatibility issue here, but any value in making the option just `connector` here? Of course we're always referring to a class, but settings like `FileStreamSink` read more like "this is the connector I want to run" rather than "this is the class of the connector I want to run".